### PR TITLE
Use alisw g4

### PR DESCRIPTION
**Pull request by benedikt-voelkel**

### CODE REVIEW

**Comment by benedikt-voelkel** [comment]
@pzhristov @ihrivnac @sawenzel

File: geant4.sh [hunk 1]
```diff
@@ -1,7 +1,8 @@
 package: GEANT4
 version: "%(tag_basename)s"
-tag: "v10.7.2-alice1"
+tag: "alice/v11.0.1-patches"
```
Comment (pzhristov) [inline geant4.sh:3]: this should be v11.0.1-alice1

File: geant4.sh [hunk 2]
```diff
@@ -1,7 +1,8 @@
...
 source: https://github.com/alisw/geant4.git
+#source: https://gitlab.cern.ch/geant4/geant4.git
```
Comment (pzhristov) [inline geant4.sh:5]: Is there any reason to keep the commented source_

Comment (pzhristov) [reply geant4.sh:5]: We can keep the line.

**Pull request merged**